crates/breez-sdk/core/src/models/mod.rs

```diff
@@ -238,12 +238,20 @@ pub enum PaymentDetails {
         invoice_details: Option<SparkInvoicePaymentDetails>,
         /// The HTLC transfer details if the payment fulfilled an HTLC transfer
         htlc_details: Option<SparkHtlcDetails>,
+        /// Conversion information if this was a successful conversion
+        conversion_info: Option<ConversionInfo>,
+        /// Conversion refund information if this was a conversion that was refunded
+        conversion_refund_info: Option<ConversionRefundInfo>,
```

Comment:
Have you considered merging these 2 into an enum?

Reply:
Done [0b0b642](https://github.com/breez/spark-sdk/pull/459/commits/0b0b6420eee3fe4a8ecc9bf6c56341c6e1d9409e)